src/request.ts

```diff
@@ -1330,38 +1323,50 @@ export type ProjectId = string | null | undefined;
 export interface RequestCallback {
   (
     a?: Error | null,
-    b?: AllocateIdsRequestResponse &
+    b?: google.datastore.v1.IAllocateIdsResponse &
```

Comment:
Hopefully I'm not opening a big can of worms, but should these response types be listed in this callback? Seems like maybe this should be a generic type? WDYT?

Reply:
It should, it is internal anyways.

Reply:
I did use `any` vs `Entity` which is masked `any` or vs extending from `GetCallback`.
I think it gives better code readability as "what you see is what you get" (personal preference).

Reply:
Sounds good to me!